pkg/system/rm_deprecated.go

```diff
@@ -0,0 +1,9 @@
+package system // import "github.com/docker/docker/pkg/system"
+
+// EnsureRemoveAll wraps `os.RemoveAll` to check for specific errors that can
+// often be remedied.
+//
+// Deprecated: moved to github.com/docker/docker/pkg/containerfs.EnsureRemoveAll()
+func EnsureRemoveAll(dir string) error {
```

Comment:
What's the purpose of this?

It's changing a compile time error to a runtime panic.

Reply:
I wanted to deprecate it; usually I would create an alias for that, but doing so would bring back in the dependency.

Do you think we should just remove it instead?

Reply:
My train of thought was that keeping it (with deprecated and a panic) would help finding the new location if anyone would be using it.

🤔 perhaps we're the only one using it though (I recall I searched, but forgot now if it was just us)

Reply:
Yes, looks to be just us using it (cri-o etc use the function, but that's from their "containers/storage" fork of docker; https://grep.app/search?q=.EnsureRemoveAll%28&case=true&filter[lang][0]=Go

I'll remove it

Reply:
done 👍